common/primitive-types/.gitignore

```diff
@@ -1,6 +1,5 @@
 /target
 **/*.rs.bk
 Cargo.lock
-bin/
```

Comment:
Let's remove wasm stuff also from gitignore

Reply:
I'd prefer a better `.gitignore` copy pasted to all the subpositions. Or a general one updated with subpaths.